ext/vscode/src/utils/azureDevCli.ts

```diff
@@ -53,13 +53,13 @@ export async function createAzureDevCli(context: IActionContext): Promise<AzureD
 export function scheduleAzdVersionCheck(): void {
     const oneSecond = 1 * 1000;
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    const minimumSupportedVersion = semver.coerce('0.8.0')!;
+    const minimumSupportedVersion = semver.coerce('0.8.0')!; // TODO: What is the new minimum version?
```

Comment:
TODO: Do

It might not need to change, lower versions of AZD should still work, they'll just do all prompts in the CLI.